i18n/README.vi-vn.md

```diff
@@ -0,0 +1,88 @@
+<p align="center">
+<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/web/static/supabase-light-rounded-corner-background.svg"/>
+</p>
+
+---
+
+# Supabase
+
+[Supabase](https://supabase.io) là một giải pháp mã nguồn mở thay thế cho Firebase. Chúng tôi sử dụng công cụ mã nguồn mở cấp doanh nghiệp để xây dựng các chức năng mà Firebase có.
+
+- [x] Cơ sở dữ liệu Postgres tự chủ
+- [x] Cập nhật dữ liệu thời gian thực
+- [x] Xác thực và phân quyền
+- [x] Tự động tạo API
+- [x] Bảng điều khiển
+- [x] Kho lưu trữ
+- [ ] Functions (sắp có)
+
+## Tài liệu
+
+Để có tài liệu đầy đủ, hãy truy cập [supabase.io/docs](https://supabase.io/docs)
+
+## Hỗ trợ & Cộng đồng
+
+- [Diễn đàn cộng đồng](https://github.com/supabase/supabase/discussions). Tốt nhất cho: hỗ trợ xây dựng và thảo luận về các phương pháp hay nhất về cơ sở dữ liệu.
+- [Vấn đề trên GitHub](https://github.com/supabase/supabase/issues). Tốt nhất cho: các lỗi và "bugs" mà bạn gặp phải khi sử dụng Supabase.
+- [Hỗ trợ email](https://supabase.io/docs/support#business-support). Tốt nhất cho: các vấn đề với cơ sở dữ liệu hoặc cơ sở hạ tầng của bạn.
+- [Discord](https://discord.supabase.com). Tốt nhất cho: chia sẻ ứng dụng của bạn hoặc dành thời gian với cộng đồng.
+
+## Trạng thái
+
+- [x] Alpha: Chúng tôi đang thử nghiệm Supabase với một nhóm người dùng kín
+- [x] Public Alpha: Bất kỳ ai cũng có thể đăng ký tại [app.supabase.io](https://app.supabase.io). Nhưng hãy bình tĩnh với chúng tôi, sẽ có một vài lỗi nhỏ.
+- [x] Public Beta: Đủ ổn định cho hầu hết các trường hợp sử dụng không phải trong môi trường doanh nghiệp (production)
+- [ ] Public: Sẵn sàng cho môi trường doanh nghiệp (production)
+
+Chúng tôi hiện đang ở giai đoạn Public Beta. Xem mục "releases" của repo này để nhận thông báo về các bản cập nhật lớn.
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Xem repo này"/></kbd>
+
+---
+
+## Cách Supabase hoạt động
+
+Supabase là sự kết hợp của các công cụ mã nguồn mở. Các tính năng của Supabase được xây dựng dựa trên các sản phẩm đạt chuẩn doanh nghiệp và mã nguồn mở. Nếu các công cụ và cộng đồng hỗ trợ công cụ đó tồn tại, cùng với giấy phép MIT, Apache 2 hoặc giấy phép mở tương đương, chúng tôi sẽ sử dụng và hỗ trợ công cụ đó. Nếu công cụ không tồn tại, chúng tôi sẽ tự xây dựng và mở mã nguồn của nó. Supabase không phải là phiên bản 1 : 1 của Firebase. Mục đích của chúng tôi là cung cấp cho các nhà phát triển trải nghiệm tuyệt vời giống như sử dụng Firebase bằng cách sử dụng các công cụ nguồn mở.
+
+**Kiến trúc hiện tại**
+
+Supabase là một [nền tảng lưu trữ cơ sở dữ liệu] (https://app.supabase.io). Bạn có thể đăng ký và bắt đầu sử dụng Supabase mà không cần cài đặt bất kỳ thứ gì. Chúng tôi vẫn đang tạo ra trải nghiệm phát triển "local" - đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng.
```

Comment:
> We are still creating the local development experience

Chung toi van dang phat trien tinh nang chay Supabase trong moi truong ao hoa tren may tinh ca nhan

@phucvo0708 what do you think?

Reply:
it's still ok if we can't improve this. So difficult to translate

Reply:
> Chung toi van dang phat trien tinh nang chay Supabase trong moi truong ao hoa tren may tinh ca nhan

Sounds great to me, I'd like to make some modifications to that too.

"_Chúng tôi vẫn đang phát triển tính năng chạy Supabase trong môi trường ảo hoá trên_ **chính** _máy tính cá nhân_ **của bạn** _- đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng._"

Just a small addition to your version, does that sounds better? @phamhieu